web-ng/src/app/components/map/map.component.ts

```diff
@@ -202,6 +215,7 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     marker.addListener('click', () => {
       this.selectMarker(marker);
       this.navigationService.setFeatureId(feature.id);
+      this.panAndZoom(options.position);
```

Comment:
...and then here use `options.position! as google.maps.LatLng` instead? The code would fail with a runtime error if position were missing or a different type, but since we are defining the value just above  in the same function, this can never happen.

Reply:
Thanks! Gentle reminder to add the cast and to change the method to accept only `google.maps.LatLng`!

Reply:
Oops!, just noticed it, sorry pushed wrong stash earlier. Fixed now.